crates/forge/bin/cmd/test/mod.rs

```diff
@@ -999,34 +991,56 @@ fn junit_xml_report(results: &BTreeMap<String, SuiteResult>, verbosity: u8) -> R
 
 #[cfg(test)]
 mod tests {
+    use crate::opts::{Forge, ForgeSubcommand};
+
     use super::*;
     use foundry_config::{Chain, InvariantConfig};
     use foundry_test_utils::forgetest_async;
 
     #[test]
     fn watch_parse() {
-        let args: TestArgs = TestArgs::parse_from(["foundry-cli", "-vw"]);
+        let args = match Forge::parse_from(["foundry-cli", "test", "-vw"]).cmd {
+            ForgeSubcommand::Test(args) => args,
+            _ => unreachable!(),
+        };
```

Comment:
This is necessary so that TestArgs has access to the global args in tests, otherwise an error like `Argument or group quiet specified in conflicts_with* for junit does not exist error` is thrown. This allows us to keep testing CLI commands rather than just the implementation. `unreachable` is required to cover the other branches, forcing a panic on a mismatch.